frontend/public/style/_common.scss

```diff
@@ -190,3 +190,21 @@ input[type=number]::-webkit-outer-spin-button {
 .co-break-word {
   @include co-break-word;
 }
+
+// append external-link icon to <a> so that it doesn't wrap without text
+// there must be no white space between the text and the closing </a> tag holding the pseudo element
+.co-external-link:after {
+  content: "\f08e";
```

Comment:
Does `$fa-var-external-link` work?

Reply:
Yeah, lines like these are totally opaque. What icon is it referencing? Who knows?